easybuild/easyconfigs/t/TopHat/TopHat-2.1.2-foss-2019b.eb

```
@@ -0,0 +1,47 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics
+# Biozentrum - University of Basel
+# Modified by: Adam Huffman
+# The Francis Crick Institute
+
+easyblock = 'ConfigureMake'
+
+name = 'TopHat'
+version = '2.1.2'
+
+homepage = 'http://ccb.jhu.edu/software/%(namelower)s/'
+description = "TopHat is a fast splice junction mapper for RNA-Seq reads."
+
+toolchain = {'name': 'foss', 'version': '2019b'}
+toolchainopts = {'cstd': 'gnu++98', 'pic': True}
+
+github_account = 'infphilo'
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+
+patches = ['%(namelower)s-2.0.13-zlib.patch']
+
+checksums = [
+    '15016b82255dad085d4ee7d970e50f0e53a280d466335553d47790d8344ff4b1',  # v2.1.2.tar.gz
+    'a33075c2affda995ddbd250933dbcaf94f68768b1eebead58a69705fc40306ef',  # tophat-2.0.13-zlib.patch
+]
+
+builddependencies = [('Autoconf', '2.69')]
+
+dependencies = [
+    ('Boost', '1.71.0', '', ('gompi', '2019b')),
```

You shouldn't need to specify this. Even if this was on foss-2019b level, it should automatically pick up dependencies from gompi, GCC, GCCcore levels.
```suggestion
    ('Boost', '1.71.0'),
```

---

Thanks for your comment. I've applied your suggested change.